spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -65,20 +70,22 @@ public String send(final String topic, Message message) throws RuntimeException
 			try {
 				return Publisher.defaultBuilder(TopicName.create(projectId, topic))
```

Comment:
Note, that the client will still have to know the project id to make it part of the `name` specification. However, hopefully that will change in the future after [this issue](https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2094) is fixed..